Mathlib/Analysis/Analytic/IsolatedZeros.lean

```diff
@@ -240,6 +240,46 @@ lemma apply_eq_zero_of_order_toNat_ne_zero (hf : AnalyticAt 𝕜 f z₀) :
   simp [hf.order_eq_zero_iff]
   tauto
 
+/-- Helper lemma for `AnalyticAt.order_mul` -/
```

Comment:
I think this should be a normal comment, and not in the final documentation: what do you think?

Reply:
I am unsure and therefore followed your suggestion. My instinct would be to mark this as 'private', but I see elsewhere that this is perhaps not the Mathlib style.